scim2-sdk-common/src/main/java/com/unboundid/scim2/common/ScimResource.java

```diff
@@ -76,11 +76,22 @@
   /**
    * Sets the schema urns for this object.  This set should contain
    * all schema urns including the one for this object and all
-   * extensions.
-   * @param schemaUrns a set containing the schema urns for this object.
+   * extensions.  The value must not be {@code null}.
```

Comment:
I added a unit test verifying this behavior, so it seemed worthwhile to add it in the documentation.